package.json

```diff
@@ -198,18 +200,16 @@
     "@babel/preset-env": "7.23.6",
     "@babel/preset-react": "7.23.3",
     "@canalplus/readme.doc": "0.5.0",
-    "@types/chai": "4.3.11",
     "@types/jest": "29.5.11",
     "@types/mocha": "10.0.6",
     "@types/react": "18.2.45",
     "@types/react-dom": "18.2.18",
     "@types/sinon": "17.0.2",
-    "@typescript-eslint/eslint-plugin": "6.16.0",
-    "@typescript-eslint/eslint-plugin-tslint": "6.16.0",
-    "@typescript-eslint/parser": "6.16.0",
+    "@typescript-eslint/eslint-plugin": "^7.11.0",
+    "@typescript-eslint/parser": "^7.11.0",
+    "@vitest/browser": "^1.6.0",
```

Comment:
Is both this one AND webdriverio required?

What does one and what does the other?